test/end-to-end/test_version_check_without_license.ps1

```diff
@@ -0,0 +1,28 @@
+Describe "version check without license" {
+    Remove-Item -Recurse -Force "$HOME/.hab/accepted-licenses"
+    Remove-Item -Recurse -Force "/hab/accepted-licenses"
+
+    It "version check without license works" {
+        hab --version
+        $LastExitCode | Should -Be 0
+    }
+
+    It "non-version check commands do not work without license" {
+        $process = Start-Process "hab" -ArgumentList "--help" -PassThru
+        {
+            $process | Wait-Process -Timeout 1 -ErrorAction Stop
+        } | Should -Throw "time-out"
+
+        $process | Stop-Process -Force
+    }
+
+    hab license accept
```

Comment:
I would put this line in the below `It` block. I forget some of the pester nuances here, but I think it may call ^^^ as it is building its list of `It` tests and therefore it could impact the top 2 tests.

Reply:
Thanks! Addressed in [2ff021](https://github.com/habitat-sh/habitat/pull/7834/commits/2ff0217633fa6e637645ba7e3c7f100191ae8052)